PF/Assignment1/anshitaarya/Ass11.java

```diff
@@ -0,0 +1,92 @@
+package anshitaarya;
+
+import java.util.Scanner;
+
+public class Ass11 {
+	
+	public static void main (String args[])
+	{
+		Scanner in = new Scanner(System.in);
+		int asize=0 ,bsize=0;
+		 System.out.println("enter size of array1 ");
+		 asize=in.nextInt();
+		int a[] = new int[asize]; 
+		
+		int i;
+		
+		 System.out.println("enter array1");
+		 
+		 for(i=0; i<asize; i++)
+		 {
+			 a[i]=in.nextInt();
+		 }
+		 System.out.println("enter size of array2 ");
+		 bsize=in.nextInt();
+		 int b[] = new int[bsize]; 
+			
+		
+		
+ System.out.println("enter array2");
+		 
+		 for(i=0; i<bsize; i++)
+		 {
+			 b[i]=in.nextInt();
+		 }
+		 int c[] = new int[asize+bsize];
+		Ass11 obj=new Ass11();
+	int[] d=obj.merge( a, asize, b,  bsize, c);
```

Comment:
ACTION: No need to pass array length as you can get length by using  array.length property.